src/psyclone/tests/domain/lfric/lfric_config_test.py

```diff
@@ -270,6 +284,37 @@ def test_default_kind():
     assert api_config.default_kind["logical"] == "l_def"
 
 
+def test_precision_map(tmpdir):
```

Comment:
This test appears only to exercise error conditions? Please rename and rephrase the docstring. Also, please could you add a separate test that we do get values read as expected.